### DISCUSSION TITLE
Adding metrics from a new application

### DISCUSSION THREAD

**Discussion by marcusnh** [prometheus-operator/kube-prometheus#1896] [Ideas]
Hi, I was wondering why there are no docs on how to add new metrics to the kube-prometheus stack. Let's say I have just implemented a new helloWorld-app in a new namespace and I want to see how it is doing.  How should I as a novice user go about doing this?
A step-by-step explaination would have been nice and will make it easier understanding how the existing stack already works.

**Comment by CoalChris** [comment]
I managed to get my application's metrics showing by adding the scraping job config as **[Additional Scrape Configuration](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/additional-scrape-config.md)**.

**Reply by theobouwman** [reply]
But this should be possible with auto discovery right?
